### DISCUSSION TITLE
Transcript timing wrong after music

### DISCUSSION THREAD

**Discussion by bmaluijb** [deepgram/community#735] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

When my user uploads new audio, we use Deepgram to transcribe it. This particular user always has intro music as the first part of their audio (which is typical, as this is a podcast episode). Almost always, this results in a time drift in the utterances that Deepgram returns. In this particular case, the drift starts around 0:40 seconds in, where it seems like there is an utterance missing. 

This does not happen for any of my other users, which often have similar audio types (podcast episodes with intro music).

Please look into this. 

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?model=nova-2&language=en-us&punctuate=true&diarize=true&ner=false&smart_format=true&utterances=true&paragraphs=true&detect_entities=false

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

68d88f7e-e4c1-4f8f-90cf-8cf3bdc09faa

### If possible, please attach an example audio file to reproduce the issue.

https://podhomemediastorage.blob.core.windows.net/media/63849972057311027586b08593-3702-4860-95f4-d631ee6d032b.mp3

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by bmaluijb** [comment]
Can someone from the Deepgram team please looks at this?

**Comment by jpvajda** [comment, marked as answer]
@bmaluijb  sorry it took us so long to respond. It looks like your attached audio file no longer available I'm getting this error:

also I noticed you applied the option `ner=false` in your request, you shouldn't need that given you applied `smart_format=true` Ner is our old approach to smart formatting and it's a deprecated API feature at this point. 


```xml
<Error>
<Code>BlobNotFound</Code>
<Message>
The specified blob does not exist. RequestId:1ab8e272-101e-0039-6b42-d2b01b000000 Time:2024-07-09T20:56:39.4411204Z
</Message>
</Error>
```
